docs/examples/query-webpage-astra-db.md

```diff
@@ -0,0 +1,73 @@
+Required Griptape extras:
+
+```
+pip install griptape[drivers-vector-astra-db,drivers-web-scraper-trafilatura]
+```
```

Comment:
Let's replace this bit with a sentence of two explaining what the example does. Then link to the AstraDb entry on `vector-store-drivers.md` which explains the extras required.

Reply:
Done (but I chose to leave a note about the `drivers-web-scraper-trafilatura` extra being also needed. Is that ok?)

Reply:
Perfect!